### PR TITLE
add install.py

### LINKED ISSUE #10: Install dependenecies in `install.py`
Webui extensions can put a file `install.py` at the root of the repo. The webui host will execute this file before starting up. It is possible to install dependencies in this file. See the docs for more info: https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Developing-extensions

For convenience, I suggest to install dependencies listed under `requirements.txt` using `install.py`.

### PR DESCRIPTION
**Pull request by ljleb**
I just copy-pasted the install script used in the controlnet extension:
https://github.com/Mikubill/sd-webui-controlnet/blob/de8fdeff373695431440b9c29e9a6fe24155e795/install.py

Closes #10

### CODE REVIEW

**Comment by s1dlx** [comment]
I’ve implement something similar here https://github.com/s1dlx/sd-webui-bayesian-merger/commit/7b83beb1598fde8cbe7e12a5307d93e165071804

but I’m happy to merge this once I’ve tested it

File: install.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import launch
+import os
+
+req_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "requirements.txt")
```
Comment (s1dlx) [inline install.py:4]: Would you mind change this to use pathlib instead of os.path? Just to be consistent with the rest of the code

Comment (ljleb) [reply install.py:4]: Yeah no problem.

**Comment by ljleb** [comment]
In fact your code seems good as it is. I'll close this as it seems to be already implemented properly on your PR.

**Pull request closed without merging**